go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/mongodb-labs/mongoreplay
+
+go 1.13
+
+require (
+	github.com/10gen/escaper v0.0.0-20160802155924-17fe61c658dc
+	github.com/10gen/llmgo v0.0.0-20180924181635-afcced55e92d
+	github.com/10gen/openssl v0.0.0-20190731121843-f50700cbcaf4
+	github.com/golang/snappy v0.0.1 // indirect
+	github.com/google/gopacket v1.1.17
+	github.com/howeyc/gopass v0.0.0-20190910152052-7cb4b85ec19c
+	github.com/jessevdk/go-flags v1.4.0
+	github.com/kr/pretty v0.2.0 // indirect
+	github.com/patrickmn/go-cache v2.1.0+incompatible
+	github.com/smartystreets/goconvey v1.6.4
+	github.com/spacemonkeygo/spacelog v0.0.0-20180420211403-2296661a0572 // indirect
+	github.com/xdg/stringprep v1.0.0 // indirect
+	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
+	golang.org/x/text v0.3.2 // indirect
+	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
+	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22
```

Comment:
at the very least, mgo will need a replace directive to point it to our fork. We should audit the old gopkg.toml to see if any others need a replace directive.

Once this file is all fixed up, we should also run `go mod vendor` to ensure the vendor dir is synced with this file.